src/components/TeamFounder/FounderProfile.styles.ts

```diff
@@ -0,0 +1,99 @@
+import styled from "styled-components/macro";
```

Comment:
Looks like this whole file is a copy of src/components/TeamSection/Profile.styles.ts

If you want to use all the same styling rules as the other profiles, then you can just import them instead.

Reply:
But we'll probably want to do some tweaks to the styling, see my comments in the Vercel preview

Reply:
Couldn't view Vercel preview. Tried signing in with my GitHub account but the error:
Authentication Error
Account not found.

Reply:
DYeah you would need to login with the tech start gmail. Anyway, my comment was to make the founder profile side by side with the description on desktop view